cli/pack/common.go

```diff
@@ -136,11 +137,11 @@ func detectVersion(ctx *context.Ctx) error {
 		switch ctx.Pack.Type {
 		case RpmType:
 			if !packageVerAllowed[RpmType].MatchString(ctx.Pack.Version) {
-				return fmt.Errorf("Dashes are not allowed in RPM --version")
+				log.Warnf("Dashes are not allowed in RPM --version")
```

Comment:
Well, if we want to loose restrictions here, we should do the same in `--suffix` validation above.

Reply:
In task author wanted to loose restrictions only for --version, I am not  sure if --suffix validation should be changed.